cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -404,6 +404,55 @@ pub enum Inst {
         offset: i64,
     },
 
+    // =====================================
+    // Instructions pertaining to atomic memory accesses.
+    /// A standard (native) `lock cmpxchg src, (amode)`, with register conventions:
+    ///
+    /// `dst`  (rd) address
+    /// `src`  (rd) replacement value
```

Comment:
nit (twice): what does `rd` mean?

Reply:
Read; I fixed all of these, and the wr and mod too.